save.el

```diff
@@ -62,3 +62,11 @@
 
 (setq yank-menu-length 9999)
 (message "→ yank-menu-length 9999!")
+
+(defun emacsti-commit-after-save ()
+       "Commit all buffers to git after save C-x C-s."
+       (interactive)
+       (shell-command-to-string
+	      "git add . && git commit -m \"Auto-commited by emacsti-commit-after-save\"")
+       (message "Exceuted → Auto-commited by emacsti-commit-after-save"))
+(add-hook 'after-save-hook #'emacsti-commit-after-save)
```

Comment:
Oi Ricardo,

Tive uma idéia de criar um alias para invocar o git-push. Dessa forma não estaríamos acionando a todo instante o repositório.

Reply:
I have tried checking if It stays in a git project using the command below.  However, I guess we could use more time before applying those features as a hook.
`git -C ~/ rev-parse; ITISGITREPO=$?; test $ITISGITREPO -eq 0 && echo " IF ITISGITREPO => $ITISGITREPO"`

Reply:
Let's add a hook or keystroke at custom.el and we can decide after some use of this solution.